### DISCUSSION TITLE
Error: PathNotFoundException: Cannot open file, path

### DISCUSSION THREAD

**Discussion by Gene-W** [KRTirtho/spotube#632] [Q&A]
Hello,
When I start spotube nothing loads and I found this in the logs:

---------- ERROR ----------
PathNotFoundException: Cannot open file, path = '/home/irish/.var/app/com.github.KRTirtho.Spotube/config/dconf/user' (OS Error: No such file or directory, errno = 2)

I didn't find anything after searching for similar issues, though I should ask for a little help with that.

**Comment by atomikaya** [comment]
Same issue. A few days ago my Spotube stopped working on all my devices at the same time which is pretty suspicious. Wondering if I got shadow banned from Spotify or something. At the same time, why would it stop me from playing from the Youtube source? Youtube on its own is fine.

There are a few errors present in my logs, so I'm not a 100% sure my problem is the same as @Gene-W exactly, but the most common one is the `config/dconf/user` file missing.

Example of spotube failing to start a song, playlist, etc:


https://github.com/user-attachments/assets/7a7eb8d5-b02a-4fbd-a439-e23b321939b8

Which doesn't start playing anything and logs nothing. The last log was on a similar attempt a few minutes prior:

```
[2024-09-26 06:35:16.976706]---------------------
PathNotFoundException: Cannot open file, path = '/home/######/.var/app/com.github.KRTirtho.Spotube/config/dconf/user' (OS Error: Aucun fichier ou dossier de ce type, errno = 2)
#0      _checkForErrorResponse (dart:io/common.dart:55)
#1      _File.open.<anonymous closure> (dart:io/file_impl.dart:381)
#2      _rootRunUnary (dart:async/zone.dart:1407)
<asynchronous suspension>
#3      _File.readAsBytes.<anonymous closure> (dart:io/file_impl.dart:562)
<asynchronous suspension>
#4      GVariantDatabase._loadRootTable (package:gsettings/src/gvariant_database.dart:30)
<asynchronous suspension>
#5      GVariantDatabase.lookup (package:gsettings/src/gvariant_database.dart:20)
<asynchronous suspension>
#6      DConfClient.read (package:gsettings/src/dconf_client.dart:82)
<asynchronous suspension>
#7      GSettingsDConfBackend.get (package:gsettings/src/gsettings_dconf_backend.dart:23)
<asynchronous suspension>
#8      GSettings.get (package:gsettings/src/gsettings.dart:124)
<asynchronous suspension>
#9      getTheme (package:titlebar_buttons/src/get_theme.dart:12)
<asynchronous suspension>
#10     _RawDecoratedTitlebarButtonState.initState.<anonymous closure> (package:titlebar_buttons/src/decorated_button.dart:156)
<asynchronous suspension>
```

Spotube is my favorite music playing app, I hope there's something I can do :smiling_face_with_tear:

**Reply by atomikaya** [reply]
Oh forgot to add a few system details:

- Version: v3.8.1 / channel stable
- Packaging: flatpak from flathub
- OS: Fedora 40 / Gnome 46 / Wayland
- Hardware: Framework 13 AMD series

**Comment by Superredstone** [comment]
I am having this issue when downloading a song named "Stabbing in The Dark" by Ice Nine Kills (All the other songs before it downloads are fine)
